components/Onboarding/OnboardingCommunity.js

```diff
@@ -121,7 +126,7 @@ function OnboardingCommunity() {
           <Link href="/onboarding/profile-selection" className={styles["back-link"]}>
             {t("back")}
           </Link>
-          <Link href="/onboarding/terms" className={styles["next-btn"]} tabIndex={6}>
+          <Link href="#" onClick={handleButtonClick} className={styles["next-btn"]} tabIndex={6}>
```

Comment:
we chose not to save it in localstorage. I know you did this so that you could implement the data update, but we can get the same result just by reusing the form that has already been created